rest_framework_json_api/utils.py

```diff
@@ -266,8 +274,11 @@ def get_included_resources(request, serializer=None):
 
 
 def get_default_included_resources_from_serializer(serializer):
+    meta = getattr(serializer, 'JSONAPIMeta', None)
+    if meta is None and getattr(serializer, 'many', False):
+        meta = getattr(serializer.child, 'JSONAPIMeta', None)
     try:
-        return list(serializer.JSONAPIMeta.included_resources)
+        return list(meta.included_resources)
```

Comment:
Slightly contrary to what I stated up above, what do you think of changing this logic to have a `hasattr` check here? I'd guess that many (most?) serializers will not have `included_resources`. Relying on an `AttributeError` for normal flow control of the majority case is bound to be slow. I recognize this would slightly increase the scope of this branch. Would you be ok fixing that while you're in here?